components/rhel/misc/rhel-k8s-singlenode-setup/README.rst

```diff
@@ -0,0 +1,31 @@
+Vagrantfile: Single Node Kubernetes
+===================================
+
+This Vagrantfile is the suggested configuration for using the CDK with Kubernetes.  This file sets up private networking that will be used to expose the docker daemon and kubernetes to the host.  This file also sets up a single node Kubernetes instance where the master and node are on the same machine.
```

Comment:
[..]expose the `docker` daemon and `kubernetes` [..]. This Vagrantfile also sets up a single node Kubernetes instance [..]